sentry/src/main/java-templates/io/sentry/environment/Version.java

```diff
@@ -0,0 +1,9 @@
+package io.sentry.environment;
+
+public final class Version {
+    public static final String SDK_VERSION = "${project.version}";
```

Comment:
Can you explain why you're using `project.version` here but it was something like `build.name` in the other plugin?  Are these two names for the same thing?

Reply:
Good catch, `project.version` is a Maven builtin and mental default for me. `buildName` is slightly different as its generated using a plugin and we include the git hash. I may as well maintain the same pattern (and use `buildName`):

https://github.com/getsentry/sentry-java/blob/27d5f6b8ec51bcf97f8d066abcb80e072652573b/sentry/pom.xml#L101-L122